parsing/dialects/dialect1/config.fmpp

```diff
@@ -1123,15 +1124,13 @@ data: {
       "InlineCaseSpecific"
       "InlineCaseSpecificNamedFunctionCall"
       "InlineModOperator"
-      "AlternativeTypeConversionLiteralOrIdentifier"
-      "InlineFormatLiteralOrIdentifier"
       "NamedLiteralOrIdentifier"
+      "AlternativeTypeConversionLiteralOrIdentifier"
     ]
 
     postExpressionMethods: [
-      "AlternativeTypeConversionQuery"
-      "InlineFormatQuery"
       "NamedQuery"
+      "AlternativeTypeConversionQuery"
```

Comment:
nit: Can we order alphabetically? If not, we should probably add a comment saying why.